_includes/configuration-db.md

```diff
@@ -36,7 +34,9 @@ disks; installation will fail if the disks are not blank.
         All {{ site.data.product.title_short }} appliances in a multi-region
         deployment must use the same key.
 
-6.  Choose **Create Internal Database** for the database location.
+6.  Choose **Create Internal Database** for the database location. 
+
+    In the **Configure Messaging** menu when you select **Make No messaging changes**, then the ```Configuration failed: Internal database require a volume mounted at /var/lib/pgsql. Please add an unpartitioned disk and try again.``` message is displayed. As a resolution, ensure to add the database disk instructions as defined above.
```

Comment:
Sorry I still think this is confusing.  `when you select **Make No messaging changes**, then the FAILURE message is displayed` insinuates that this is going to happen every time.

Prefer something like:
```suggestion
    In the **Configure Messaging** menu select **Make No messaging changes**.  If you see ```Configuration failed: Internal database require a volume mounted at /var/lib/pgsql. Please add an unpartitioned disk and try again.``` displayed make sure to add a second disk for the database per instructions defined above.
```

Reply:
Updated as suggested, please check and confirm. TIA